src/renderer/store/localStore.ts

```diff
@@ -1,11 +1,16 @@
 import ElectronStore from 'electron-store';
+import packageJson from '../../../package.json';
 
-const localStore = new ElectronStore({
-  migrations: {
-    '1.0.0-alpha.26': (store) => {
-      store.set('sockets', {});
-    },
-  },
-});
+const migrations: any = {};
+const projectVersion = packageJson.version;
```

Comment:
Hey @charnysh-dev can you see if this works for us:-

import {remote} from 'electron';
const projectVersion = process.env.npm_package_version || remote.app.getVersion();

Because I think if we import something from package.json, the file will become the part of our build.

Reply:
@lapstjup Great, it looks professional. Got it fixed.